Month2/week1/Month2-Project Week 1-Remembering the Bases/VideoGame/VideoGames.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include<iostream>
+#include <string>
+
+class FVideoGame
+{
+	private:
+		std::string GameName;
+		std::string	StudioName;
+		int DayCreation;
+		int MonthCreation;
+		int YearCreation;
+	public:
+		FVideoGame();
+		FVideoGame(const std::string NewGameName, const std::string NewStudioName, const int NewDay, const int NewMonth, const int NewYear);
+		std::string GetGameName()const;
+		std::string GetStudioName()const;
+		int GetDayCreation()const;
+		int GetMonthCreation()const;
+		int GetYearCreation()const;
+		std::string SetGameName(std::string NewGameName);
+		std::string SetStudioName(std::string NewStudioName);
+		int SetDayCreation(const int NewDayCreation);
+		int SetMonthCreation(const int NewMonthCration);
+		int SetYearCreation(const int NewYearCreation);
+		std::string GetFormatedDate()const;
+		bool GameIsValid()const;
```

Comment:
The game is redundant. The class is already called FVideoGame, so a simple IsValid would be enough.